dotcom-rendering/src/components/LiveBlogEpic.importable.tsx

```diff
@@ -260,6 +271,7 @@ export const LiveBlogEpic = ({
 		keywordIds,
 	});
 	if (!payload) return null;
+	if (shouldRemoveEpic) return null;
```

Comment:
Minor: we could move this immediately below the line that sets it. Returning early avoids the need to construct the payload if we're not going to use it.

Reply:
Good catch! Will do that

Reply:
I think either of the two options you suggest Emma would be fine